.github/workflows/reusable_lint.yml

```
@@ -0,0 +1,24 @@
+name: "Lint Packages"
+
+on: 
+  workflow_call:
+    secrets:
+      THE_GH_RELEASE_TOKEN:
+        required: true
+
+jobs:
+  lint:
+    runs-on: 'macos-12'
```

From what I saw [inappbrowser](https://github.com/ionic-team/capacitor-os-inappbrowser/pull/38), a more recent version for mac runner is needed.

```suggestion
    runs-on: 'macos-14'
```

---

Using `macos-latest` probably the best option to avoid the same issue